sdk/src/feature_set.rs

```diff
@@ -687,6 +692,7 @@ lazy_static! {
         (cap_transaction_accounts_data_size::id(), "cap transaction accounts data size up to its compute unit limits #27839"),
         (enable_alt_bn128_syscall::id(), "add alt_bn128 syscalls #27961"),
         (enable_program_redeployment_cooldown::id(), "enable program redeployment cooldown #29135"),
+        (prevent_commission_update_in_second_half_of_epoch::id(), "prevent updating validator commission in the second half of an epoch"),
```

Comment:
Flipping the description around feels a little more natural to me
```suggestion
        (commission_updates_only_allowed_in_first_half_of_epoch::id(), "validator commission updates are only allowed in the first half of an epoch"),
```

Reply:
Will do, I went back and forth while implementing it, as you probably noticed by this inconsistency.